test/feature/reverse_routing.feature

```diff
@@ -0,0 +1,31 @@
+Feature: reverse routing
+
+  Background:
+    Given an empty "controllers" directory
+
+
+  Scenario: routes can define reverse routes with 'as'
+    Given a file "routes.coffee" with the contents
+      """
+      module.exports = ({ GET }) ->
```

Comment:
where would it get `some_route` from? The reverse route shouldn't change when changing the url